res/controllers/Reloop Terminal Mix 2-4.js

```diff
@@ -121,6 +121,11 @@ TerminalMix.samplerVolume = function (channel, control, value) {
     }
 }
 
+TerminalMix.pitchSlider = function (channel, control, value, status, group) {
+	// invert pitch slider (down=faster) so it matches the labels on controller
+	engine.setValue(group,"rate",-script.midiPitch(control, value, status));
+}
+
```

Comment:
In the future, please open a separate pull request for independent changes. You can do this by checking out the master branch, then creating a new branch from there. If you don't plan on changing anything else in the Reloop Terminal Mix script, it's probably not worth the trouble to separate this commit from the others in this PR.